Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "3.9"
+
+[dev-packages]
+pre-commit = "==2.*"
+sceptre = "==2.*"
```

Comment:
No, not really. Let's say you have a lambda repository: there are going to be things in it that should be legitimately in `[packages]` b/c the lambda depends on them.

On the other hand an argument can be made that this repository should be title `sceptre-infra-repository`, then all these things could just go in packages. I have placed them in `[dev-packages]` because @zaro0508 previously expressed strong feelings about this.